monkey/monkey_island/cc/models/monkey.py

```diff
@@ -42,9 +46,18 @@ def get_single_monkey_by_id(db_id):
         except IndexError:
             raise MonkeyNotFoundError("id: {0}".format(str(db_id)))
 
+    @staticmethod
+    def get_single_monkey_by_guid(monkey_guid):
+        try:
+            return Monkey.objects(guid=monkey_guid)[0]
+        except IndexError:
+            raise MonkeyNotFoundError("guid: {0}".format(str(monkey_guid)))
```

Comment:
Hmmm... Can we have 2 monkey objects with the same guid? Also, you might want to look at this:
![image](https://user-images.githubusercontent.com/36815064/61638969-3035f500-aca3-11e9-8b9a-d2f83735d640.png)

Reply:
Fixed, good catch, it does look better.